docs/source/build-run-openvino.md

```diff
@@ -0,0 +1,202 @@
+::::{grid} 2
+:::{grid-item-card}  What you will learn in this tutorial:
+:class-card: card-prerequisites
+* In this tutorial you will learn how to lower and deploy a model with OpenVINO.
+:::
+:::{grid-item-card}  Tutorials we recommend you complete before this:
+:class-card: card-prerequisites
+* [Introduction to ExecuTorch](intro-how-it-works.md)
+* [Setting up ExecuTorch](getting-started-setup.md)
+* [Building ExecuTorch with CMake](runtime-build-and-cross-compilation.md)
+:::
+::::
+
+## Introduction to OpenVINO
+
+[OpenVINO](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html) is an open-source toolkit designed to enhance AI inference on Intel hardware by reducing latency and increasing throughput while preserving accuracy. It optimizes hardware utilization and simplifies AI development and deep learning integration across domains such as computer vision, large language models (LLMs), and generative AI.
+
+OpenVINO is integrated as an Executorch delegate to accelerate AI applications deployed with Executorch APIs. 
+
+## Supported Hardware
+
+OpenVINO backend supports the following hardware:
+
+- Intel CPUs
+- Intel integrated GPUs
+- Intel discrete GPUs
+- Intel NPUs
+
+## Directory Structure
+
+```
+executorch
+├── backends
+│   └── openvino
+│       ├── runtime
+│           ├── OpenvinoBackend.cpp
+│           └── OpenvinoBackend.hpp
+│       ├── scripts
+│           └── openvino_build.sh
+│       ├── tests
+│       ├── CMakeLists.txt
+│       ├── README.md
+│       ├── __init__.py
+│       ├── openvino_functions.yaml
+│       ├── partitioner.py
+│       ├── preprocess.py
+│       └── requirements.txt
+└── examples
+│   └── openvino
+│       ├── aot
+│           ├── README.md
+│           └── aot_openvino_compiler.py
+│       └── executor_runner
+│           └── openvino_executor_runner.cpp
+│       ├── CMakeLists.txt
+│       ├── README.md
+└──     └── openvino_build_example.sh
+```
+
+## Instructions for Building OpenVINO Backend
+
+### Prerequisites
+
+Before you begin, ensure you have openvino installed and configured on your system:
+
+#### TODO: Update with the openvino commit/Release tag once the changes in OpenVINO are merged
+#### TODO: Add instructions for support with OpenVINO release package
+
+```bash
+git clone -b executorch_ov_backend https://github.com/ynimmaga/openvino
+cd openvino
+git submodule update --init --recursive
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release -DENABLE_PYTHON=ON -DENABLE_WHEEL=ON
+make -j<N>
+
+cd ../..
+cmake --install build --prefix <your_preferred_install_location>
+cd <your_preferred_install_location>
+source setupvars.sh
+```
+
+### Setup
+
+Follow the steps below to setup your build environment:
+
+1. **Setup ExecuTorch Environment**: Refer to the [Setting up ExecuTorch](https://pytorch.org/executorch/stable/getting-started-setup) guide for detailed instructions on setting up the ExecuTorch environment.
```

Comment:
Same comment applies as in the "backends/openvino/README.md" file. Should we specify using instructions from "Environment Setup"?

Reply:
Done